RecoLocalTracker/SiStripClusterizer/python/SiStripClusterizer_RealData_cfi.py

```diff
@@ -11,6 +11,11 @@
     cms.InputTag('siStripZeroSuppression','ScopeMode')),
                                )
 
+from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
+from RecoLocalTracker.SiStripClusterizer.SiStripApprox2Clusters_cfi import SiStripApprox2Clusters
+SiStripApprox2Clusters.inputApproxClusters = 'SiStripClusters2ApproxClusters'
+approxSiStripClusters.toReplaceWith(siStripClusters,SiStripApprox2Clusters)
```

Comment:
This modification should be in approxSiStripClusters_cff.py instead

Reply:
Sorry, but also this comment of mine was wrong. `siStripClusters` was actually defined in this file (RecoLocalTracker/SiStripClusterizer/python/SiStripClusterizer_RealData_cfi.py) and therefore this was already the right place where to modify it. Could you please revert, (and blame me for the mistake)?